src/assets/styles/components/_tooltip.scss

```diff
@@ -0,0 +1,30 @@
+.tooltip-class {
+    z-index: 1000;
+}
+
+.tooltip-inner {
+    background-color: #f3f4f6;
+    position: relative;
+    font-size: 12px;
+    border: 1px solid #ced4da;
+    padding: 5px 10px;
+    color: black;
+    text-align: left;
+    width: 150px;
+    margin-bottom: 5px;
+}
+
+.bs-tooltip-top,
+.bs-tooltip-bottom {
+    .tooltip-arrow {
+        &::before {
+            border-top-color: #ced4da !important;
+            border-bottom-color: #ced4da !important;
+            top: -5px !important;
```

Comment:
I think it should work without `!importants` - I checked a few places, and it was OK.
But I could miss something. If there are any style collisions - please contact me, and we could try to solve them together

Reply:
Yes, thanks. Removed